test/blackbox-tests/test-cases/toplevel-plugin-fail.t

```diff
@@ -0,0 +1,184 @@
+Testsuite for (toplevel that loads plugins). This version
+uses ``dune-site.dynlink`` which uses ``Dynlink.loadfile``.
+This is not allowed in toplevels, so it fails.
+
+  $ cat > dune-project <<EOF
+  > (lang dune 3.7)
+  > (using dune_site 0.1)
+  > (name top_with_plugins)
+  > (wrapped_executables false)
+  > (map_workspace_root false)
+  > 
+  > (package
+  >   (name top_with_plugins)
```

Comment:
We indent dune files with a single space only.

Reply:
> We indent dune files with a single space only.

I will fix these.